index.html

```diff
@@ -32,31 +32,168 @@ <h1 class="profile">I'm Damilare Glad to see you!</h1>
+                        <div class="btn">html</div>
+                        <div class="btn">css</div>
+                        <div class="btn">javascript</div>
+                    </div>
+                    <button type="button" class="bon">See Project</button>
+                </li>
+                <li class="project-card">
+                    <img src="./images/project-3.png" class="pix" alt="snap">
+                    <h1 class="project-title">Multi-Post Stories</h1>
+                    <p class="project-description">CANOPY
+                        <span><img src="./images/rename.png" alt="period">Back End Dev </span>
+                        <span><img src="./images/rename.png" alt="period">2015</span>
+                    </p>
+                    <p class="project-content">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="lang">
+                        <div class="btn">html</div>
+                        <div class="btn">css</div>
+                        <div class="btn">javascript</div>
+                    </div>
+                    <button type="button" class="bon">See Project</button>
+                </li>        
+           </ul> 
+        </section>
+        <section class="myself">
+            <div class="about">
+                <h1 class="about-myself">About Myself</h1>
+                <p class="myself-content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p><a href="#" class="connect">LET'S CONNECT</a></p>
+                <ul>
+                    <li>
+                        <a href="#">
+                           <img src="./images/./Twitter-logo.png" alt="Twitter-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                          <img src="./images/Linkedin-logo.png" alt="Linkedin-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                         <img src="./images/M-logo.png" alt="letter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                          <img src="./images/Github-logo.png" alt="Github-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                          <img src="./images/fingers-logo.png" alt="fingers-logo">
+                        </a>
+                    </li>
+                </ul>
+                <button type="button" class="resume">Get my resume</button>
+            </div>
+            <div class="container">
+                <div>
```

Comment:
- [ ] Instead of `div` tag a `ul` tag would wrap the technology categories (language, frameworks, skill) as `li` elements. Kindly fix this.